### PR TITLE
Add multi-argument and options example

### PR DESCRIPTION
**Pull request by justsml**
I felt this was another missing example.

I gathered some feedback from some of my students & friends who use `execa` and I saw some common themes. Planning on putting a few minimal examples together.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -41,6 +41,28 @@ const execa = require('execa');
...
+
+Let's take a look at how we can accomplish the same using `execa`:
+
+```js
+const execa = require('execa');
+
+(async () => {
+	const options = {env: {NODE_ENV: 'production'}};
+	const {stdout, stderr} = await execa('npm', ['test', '--', '--json'], options);
+	console.log({stdout, stderr});
```
Comment (ehmicky) [inline readme.md]: Indentation should be 3 spaces like the rest of the `README` for consistency.

Ideally indentation should be 2 or 4 spaces across the `README` because 3 is weird, but that should be a separate PR :)

File: readme.md [hunk 2]
```diff
@@ -41,6 +41,28 @@ const execa = require('execa');
...
+})();
+```
+Notice how the arguments are specified as array elements: `['test', '--', '--json']`.
```
Comment (ehmicky) [inline readme.md]: There should be a blank line before this line.

File: readme.md (hunk 1 above)
Comment (justsml) [reply readme.md]: I did it using the online editor 🤔 ... Will fix

Comment (justsml) [reply readme.md]: Hmmm, are you sure? 
It looks tab indented like the rest to me 🙃

File: readme.md (hunk 2 above)
Comment (justsml) [reply readme.md]: It's showing fixed under this specific commit: https://github.com/sindresorhus/execa/pull/387/commits/131bfa545482f63cac6fb4d4357e948e01379d72

But not under the main PR view 🤷‍♂ 

Maybe a github caching issue. Let me know if it's still broken for you.

Comment (ehmicky) [reply readme.md]: The line is here :)

File: readme.md (hunk 1 above)
Comment (ehmicky) [reply readme.md]: Yes sorry, you're correct!

**Comment by ehmicky** [comment]
Would it be possible to either:
  - merge this example into the same code block as the other examples
  - break down the other examples into separate code blocks with their own sub-headers

Otherwise the example section is mixing both of those styles. Also that new example should probably be last. Thanks for your work on this!

**Comment by ehmicky** [comment]
@justsml If you would like some help with completing the PR, please let me know, I can help :)

**Comment by sindresorhus** [comment]
Closing for lack of activity.
